src/containers/quiz/quiz-question/TutorAnswerGrading.tsx

```diff
@@ -0,0 +1,154 @@
+import { Box, IconButton, Stack } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import CheckIcon from '@mui/icons-material/Check'
+import CloseIcon from '@mui/icons-material/Close'
+import { styles } from '~/containers/quiz/quiz-question/Question.styles'
+import { useCallback, useEffect, useState } from 'react'
+import { ResourceService } from '~/services/resource-service'
+import { useParams } from 'react-router-dom'
+import useQuery from '~/hooks/use-query'
+import { ONE_HOUR, snackbarVariants } from '~/constants'
+import useMutation from '~/hooks/use-mutation'
+import { type ErrorResponse } from '~/types'
+import { useAppDispatch } from '~/hooks/use-redux'
+import { openAlert } from '~/redux/features/snackbarSlice'
+import { getErrorKey } from '~/utils/get-error-key'
+
+interface TutorAnswerGradingProps {
+  questionText?: string
+  onUpdate?: (isCorrect: boolean) => void
+}
+
+const TutorAnswerGrading: React.FC<TutorAnswerGradingProps> = ({
```

Comment:
```suggestion
const TutorAnswerGrading: FC<TutorAnswerGradingProps> = ({
```
And import this FC from 'react'

Reply:
actually no, on the whole project we use React.FC so it's supposed to be like that for consistency

Reply:
Okey i get it
I just thought I saw something like this in this files, so that is why I wrote it